### ISSUE TITLE
pageshot.net lacks caching for all static assets

### ISSUE THREAD

**Issue by peterbe** [mozilla-services/pageshot#1624]
All repeat views take the same amount of time as the first view
https://www.webpagetest.org/result/160929_M4_1BQB/

**Comment by ianb** [comment]
There's some notes about this in https://github.com/mozilla-services/pageshot/issues/1557 about this – I think it's not *all* the static assets, but a bunch of them

**Comment by peterbe** [comment]
16 files that are missing it according to webpagetest.org.

**Comment by peterbe** [comment]
Also, when the cache headers are fixed, moving them to a CDN would also be great.

**Comment by ianb** [comment]
I've added #1801 and #1802 to decrease the number of static assets slightly.

All resources should now have caching.  ga-activation.js is somewhat of a special case, it's still cached but private (and can't go on a CDN).

We have minimization now running on the bundles (where most page logic goes) which decreases their size by about 50%.  The bundles are still averaging 500Kb, so we could probably look more closely at the dependencies to decrease that.  `build/dependency-cache/` has listings.  It's almost entirely React, so we can look at React-specific bundle optimizations.  Followup in #1803

Making more scripts async is in #1804

We also put some content inline into the script that we don't have to, we could serve up a page that isn't live, and hydrate it in a subsequent request.  That is listed in #1805.

Closing this ticket in preference of those specific tickets.
